soc/intel/intel_adsp/ace/Kconfig

```diff
@@ -13,16 +12,20 @@ config SOC_SERIES_INTEL_ADSP_ACE
 	select SOC_HAS_RUNTIME_NUM_CPUS
 	select HAS_PM
 
-config SOC_INTEL_ACE15_MTPM
-	select SOC_SERIES_INTEL_ADSP_ACE
-
-config SOC_INTEL_ACE20_LNL
-	select SOC_SERIES_INTEL_ADSP_ACE
-
 config SOC_INTEL_COMM_WIDGET
 	bool "Intel Communication Widget driver"
 	default y
 	depends on DT_HAS_INTEL_ADSP_COMMUNICATION_WIDGET_ENABLED
 	help
 	  Select this to enable Intel Communication Widget driver.
 	  DSP Communication Widget is a device for generic sideband message transmit/receive.
+
+config SOC_INTEL_ACE15_MTPM
+	bool
+	help
+	  ACE 1.5 Meteor Lake PCH M
+
+config SOC_INTEL_ACE20_LNL
+	bool
+	help
+	  ACE 2.0 Lunar Lake PCH
```

Comment:
why ?
Those are defined with `bool`s here:
https://github.com/zephyrproject-rtos/zephyr/blob/d87086055b5fa44d951e6e9a8b9f87c691a39af5/soc/intel/intel_adsp/ace/Kconfig.soc#L11-L17

Reply:
Will address post-rebase to unblock vendors